src/View/Exasol/ExasolViewReflection.php

```diff
@@ -59,7 +59,9 @@ public function getViewDefinition(): string
             ExasolQuote::quote($this->viewName)
         );
 
-        return $this->connection->fetchOne($sql);
+        /** @var string $definition */
+        $definition = $this->connection->fetchOne($sql);
```

Comment:
neni tady potreba to pretypovat nebo assertnout?

Reply:
Přetypovat to nemusíš, protože VIEW_TEXT je string.
Každopádně assert možná, jo. Asi to vrátí `''` nebo null když to view neexistuje.
Každopádně to není účel tady toho PR opravit všecko na co sa dojde, takže bych to viděl spíš na story.

Reply:
opravil sem taky tady https://github.com/keboola/php-table-backend-utils/pull/60